src/components/ActionsBuilder/SupportedActions/index.tsx

```diff
@@ -89,7 +89,7 @@ export const supportedActions: Record<
     infoLineView: UpdateENSContentInfoLine,
     summaryView: UpdateENSContentSummary,
     editor: UpdateENSContentEditor,
-    displaySubmit: true,
+    displaySubmit: false,

```

Comment:
This was the last action that handle a submit button from ActionsModal component. I think this is the right moment to remove the Submit Button from ActionsModal ( https://github.com/DXgovernance/DAVI/blob/develop/src/components/ActionsModal/ActionsModal.tsx#L167) and all the displaySubmit logic.